README.md

```diff
@@ -39,16 +39,28 @@ emacs -Q --batch --eval '(princ (with-temp-buffer (insert-file-contents "file.tx
 
 ### Pipe league
 
+- Needs pipes to work.
+
 ```sh
 tee < file.txt
 tr a a < file.txt
 rev file.txt | rev
 echo ',p' | ed -s file.txt
 ```
 
+### Brittle league
+
+- May not handle every character.
+- May not handle every file length.
+
+```sh
+sed '/*/p' file.txt
```

Comment:
As there is a sed one in pure league does it make sense to list this one? i'll merge and we can remove later if so